templates/ts-apollo-mongodb-backend/README.md

```diff
@@ -1,35 +1,13 @@
-## Graphback Apollo+MongoDB Runtime Template
+## Graphback MongoDB + Apollo Client Runtime Template
 
-Template showcases Graphback Runtime capabilities using
-resolver layer created in-memory as opposed to codegeneration.
+A template which showcases Graphback Runtime capabilities using a resolver layer created in-memory as opposed to codegeneration.
 
-Serverless example can be build as docker image with configurable volume for models. 
-This models will be processed at application at startup giving fully featured GraphQL Server 
-following Graphback CRUD specification.
+Serverless example can be build as docker image with configurable volume for models.
+These models will be processed in the application at startup giving a fully featured GraphQL Server following Graphback CRUD specification.
```

Comment:
This section does not make much sense any more, could you see about bringing your own take on it?

Reply:
I would definitely like to reword it since I myself don't understand it.

1. You mentioned above that Graphback does not generate GraphQL resolvers for the server anymore and is a "runtime-only server solution". What I didn't understand here is what a runtime server solution is? If the GraphQL resolvers are not being generated by Graphback then how exacly is the server working and adapting to the changes in the business model made by the user? 
2. I don't know much about Docker except that it is used so that we don't run into problems when running code on different machines. And as far as I could understand Docker is being used here to start the database so that nobody has any problem running it regardless of whether they have mongodb or postgres installed locally, right? But what does building a "Serverless example" as a docker image mean?

Reply:
Great questions.

What I mean to say is that Graphback does not generate resolver **files** - it does generate resolvers which are completely in memory and are output as `resolvers` in [`buildGraphbackAPI`](https://graphback.dev/docs/api/graphback/interfaces/_buildgraphbackapi_.graphbackapi#resolvers).

You can actually completely remove the paragraph about serverless, I see no relevance to the current template, and it will just confuse people :D

Serverless Architecture is when you use cloud services to run your code without managing servers - so it is definitely out of the scope of this template.

Reply:
Thanks for clearing stuff out :)